package.json

```diff
@@ -93,10 +93,10 @@
     "liquid-fire": "0.26.4",
     "loader.js": "^4.0.10",
     "morgan": "^1.6.1",
-    "nypr-account-settings": "nypublicradio/nypr-account-settings",
     "nypr-ads": "nypublicradio/nypr-ads",
     "nypr-player": "nypublicradio/nypr-player",
     "nypr-ui": "nypublicradio/nypr-ui",
+    "nypr-account-settings": "nypublicradio/nypr-account-settings#d4a20e76df91c1150e581a16bcf47420b38d98d8",
```

Comment:
after merging this https://github.com/nypublicradio/nypr-account-settings/pull/8 i should be able to remove the hash and this and https://github.com/nypublicradio/wqxr-web-client/pull/73 can be merged